### PR TITLE
Add add-on message documentation

### PR DESCRIPTION
**Pull request by MattLichtenstein**
## Description

Wiki documentation for add-on messages

## Reference

None

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: docs/messages.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Messages
+
+Messages, located in the Messages Inbox, are used to notify users about important information
```
Comment (lesleyjanenorton) [inline docs/messages.md:3]:
```suggestion
Messages, located in the Messages Inbox, are used to notify users about important information.
```

File: docs/messages.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The Message JSON files are add-on JSON objects, with type "message" and a "message"
+property object with the following properties:
+
+| Property | Description  | Type | Required |
+| -- | -- | --| -- |
+| id | A unique identifier for the message | String | Yes |
+| title | The subject of the message  | String | Yes |
+| subtitle | A brief description of the message | String | No |
+| title_comment | An optional comment to describe the meaning of the title | String | No |
```
Comment (lesleyjanenorton) [inline docs/messages.md:22]: Is `title_comment` used exclusively by l10n?

**Review by lesleyjanenorton — APPROVED** [review]

File: docs/messages.md (hunk 2 above)
Comment (bakulf) [reply docs/messages.md:22]: Yes and we should have subtitle_comment too

File: docs/messages.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+| date | The date the message was received | Number | No |
```
Comment (bakulf) [inline docs/messages.md:23]: Let's write here that it is a epoch time in seconds.

**Review by bakulf — APPROVED** [review]

**Pull request merged**